loginsightwebhookdemo/bugzilla.py

```diff
@@ -81,4 +88,10 @@ def bugzilla(ALERTID=None, TOKEN=None, PRODUCT=None, COMPONENT=None, VERSION=Non
             "summary" : a['AlertName'],
             "description": a['info'],
         }
-        return callapi(BUGZILLAURL + '/rest/bug?' + auth, 'post', json.dumps(payload), headers, None, check = VERIFY)
+        # op_sys and rep_platform may not be needed, but are to test on landfill
+        if "landfill.bugzilla.org" in BUGZILLAURL:
```

Comment:
Indeed -- I consider this a poor's man approach to validating the URL. Trying to prevent the user from doing something undesired with a cheap check.